recorded-talks.md

```diff
@@ -14,6 +14,7 @@ redirect_from:
 Full archives of 2018 videos are available:
 
 - IPFS: [ipfs://QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6]({{ site.ipfs_gateway }}/QmQQfD1ztg1aG82VmUmutusbxn7tL4c2o1qHv1ivyWbNj6/)
+- Mirror: [Toronto Mesh Alexandria Mirror](https://alexandria.tomesh.net/ournetworks/2018)
```

Comment:
I think the mirror part may be confusing. It mirror Freifunk content into a directory, but it's a regular media host for the Our Networks videos. @DarkDrgn2k any idea how we should refer to Alexandria?

My suggestion would be:

```suggestion
- Toronto Mesh: [Alexandria Media Host](https://alexandria.tomesh.net/ournetworks/2018) (cjdns & yggdrasil accessible)
```

Reply:
I think we need more info if we want to mention it being "cjdns / ygg available".... like... how? is there a way that info is shared and documented by Tomesh anywhere we can link to?